taqueria-vscode-extension/src/extension.ts

```diff
@@ -33,7 +35,7 @@ export async function activate(context: api.ExtensionContext, input?: InjectedDe
 	// Third-party plugins aren't exposed via the VS Code interface
 	if (folders.length === 1) {
 		await makeDir(folders[0].uri.path, i18n)
-			.then(projectDir => {
+			.then(async projectDir => {
```

Comment:
Why did we add `async` when there's no `await` in the function?

Reply:
Yes, this is a remnant of an older change. Now that nothing is async, I can revert this change.